process/process.go

```diff
@@ -18,6 +18,7 @@ func init() {
 
 type Process struct {
 	Pid            int32 `json:"pid"`
+	NSPid          int32 `json:"nspid"`
```

Comment:
Does it have to a public field? I guess a better question is: is it in any part of json parsing?

Reply:
In the `Process.String()` method, we use the `json.Marshal` function. I added the json annotation to keep it consistent with the `Pid` field. 

https://github.com/DataDog/gopsutil/blob/7b111bf43e048e58475ffb3fda9e81a8365dbe53/process/process.go#L107-L110